.github/workflows/pull_request.yml

```
@@ -1,6 +1,17 @@
 name: Build Chain
 
-on: [pull_request]
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - 'LICENSE*'
+      - '.gitattributes'
+      - '.gitignore'
+      - '*.md'
+      - '*.adoc'
+      - '*.txt'
+      - 'runLocally.sh'
+      - 'runOnOpenShift.sh'
```

`.ci/**` ?

---

you are right, adding it
```suggestion
      - 'runOnOpenShift.sh'
      - '.ci/**'      
```